src/js/tracks/html-track-element.js

```diff
@@ -0,0 +1,81 @@
+import * as browser from '../utils/browser.js';
+import EventTarget from '../event-target';
+
+/**
+ * https://html.spec.whatwg.org/multipage/embedded-content.html#htmltrackelement
+ *
+ * interface HTMLTrackElement : HTMLElement {
+ *   attribute DOMString kind;
+ *   attribute DOMString src;
+ *   attribute DOMString srclang;
+ *   attribute DOMString label;
+ *   attribute boolean default;
+ *
+ *   const unsigned short NONE = 0;
+ *   const unsigned short LOADING = 1;
+ *   const unsigned short LOADED = 2;
+ *   const unsigned short ERROR = 3;
+ *   readonly attribute unsigned short readyState;
+ *
+ *   readonly attribute TextTrack track;
+ * };
+ */
+
+class HTMLTrackElement extends EventTarget {
+  constructor(track) {
+    super();
+
+    var readyState,
+        textTrack,
+        trackElement = this;
+
+    if (browser.IS_IE8) {
+      // TODO: do stuff
+    }
+
+    this.kind = track.kind;
+    this.src = track.src;
+    this.srclang = track.srclang;
+    this.label = track.label;
+    this.default = track.default;
+
+    const NONE = 0;
```

Comment:
Move the constants outside of `HTMLTrackElement` and export them?—Is this what you mean?